apps/studio/stories/Checkbox.stories.tsx

```diff
@@ -1,34 +1,159 @@
 import { Meta, StoryObj } from "@storybook/react";
 
-import { Checkbox, Label } from "@kampus/ui";
-
-const labels = ["köpekleri çıkarmak", "alışverişe gitmek", "kitap okumak"];
+import { Button, Checkbox, Form, FormControl, FormDescription, FormField, FormItem, FormLabel, Label, toast, useForm, z } from "@kampus/ui";
+import { Link } from "lucide-react";
+import { zodResolver } from "@hookform/resolvers/zod"
 
 const meta = {
   title: "Checkbox",
+  tags: ["autodocs"],
   component: Checkbox,
+  argTypes: {
+    disabled: {
+      control: "boolean",
+    },
+    checked: {
+      control: "boolean",
+    }
+  },
+
 } satisfies Meta<typeof Checkbox>;
 
 export default meta;
-type NewType = StoryObj<typeof meta>;
+type Story = StoryObj<typeof meta>;
+
+export const DefaultChecked = {
+  args: {
+    defaultChecked: true,
+  },
+} satisfies Story;
+
+export const OnFocus = {
+  args: {
+    onFocus: () => {
```

Comment:
lets not manually manipulate DOM elements. Instead we can use onFocus and onBlur properties of the checkbox element for changing the text of the label inside.